reference/progress.v1.yaml

```diff
@@ -312,316 +162,18 @@ paths:
       description: Get simple progress statements for a specific user
 components:
   schemas:
-    Statement:
-      title: XAPI Statement
+    SimpleProgress:
+      title: SimpleProgress
       type: object
       x-examples:
         example-1:
           id: string
```

Comment:
Is there a usefull value for this? Shouldnt we use the combination of (Activity/AU)/Student

Reply:
Ik zou een gui verwachten. kun je later kijken of je deze event/record al geprocessed hebt.